src/Tools/ExternalAccess/Razor/IRazorDocumentOptionsService.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor
+{
+    using System.Threading;
```

Comment:
Nit: Seems like the style of the rest of this is using outside the namespace.

Reply:
Thanks for the catch! You're right, I moved them up.